checksums.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -u
+set -e
+
+echo "###########################################"
+echo "############ Checksum of docker image #####"
+echo "###########################################"
+
+grep "Digest: sha256:" .qsub.out
```

Comment:
There is no such file named `.qsub.out`. You are missing a * to match all files starting with `.qsub.out`.